completion/tests/test_utilities.py

```diff
@@ -0,0 +1,50 @@
+'''
+The unit tests for utilities.py.
+'''
```

Comment:
Hmm, I'm surprised the travis build passes without a `from __future__ import unicode_literals` in here - there's usually a python 3 linting error if it's not included.